.prettierrc

```diff
@@ -0,0 +1,15 @@
+{
+  "overrides": [
+    {
+      "files": "*.sol *.js",
+      "options": {
+        "printWidth": 120,
```

Comment:
I think 140 is better.

Reply:
Updated to 140 in the latest commit aa878fd